### PR TITLE
Fix appveyor test failures

### PR DESCRIPTION
**Pull request by christianpaquin**
Appveyor installed a newer version of MSVC (from 19.26.28806.0 to 19.27.29111.0), which resulted in failures on the nightly build. This PR tweaks the picnic include files to accommodate this change.

### CODE REVIEW

**Comment by dstebila** [comment]
Rebase off of master once #822 lands and that should resolve the Doxygen failure in the macOS-shared-openssl build.

**Comment by christianpaquin** [comment]
> Rebase off of master once #822 lands and that should resolve the Doxygen failure in the macOS-shared-openssl build.

We might have a little chicken and egg problem here: it looks like #822 is blocked by the appveyor failure that this PR fixes. Perhaps we should include this commit in the other PR.

**Comment by dstebila** [comment]
> > Rebase off of master once #822 lands and that should resolve the Doxygen failure in the macOS-shared-openssl build.
> 
> We might have a little chicken and egg problem here: it looks like #822 is blocked by the appveyor failure that this PR fixes. Perhaps we should include this commit in the other PR.

Okay, if the CI on the other PR passes except for AppVeyor, I'll cherry pick this over to that PR.

**Review by dstebila — APPROVED** [review]

**Pull request merged**
